src/java_binding/make-java-binding.toml

```diff
@@ -0,0 +1,41 @@
+[tasks.gen-java-binding-header]
+description = "Generate the java binding C header file"
+script = '''
+#!/bin/bash
+set -ex
+cd src/java_binding/java
+javac -h . -cp java-binding/src/main/java/ java-binding/src/main/java/com/risingwave/java/binding/Binding.java
+'''
+
+[tasks.build-java-binding-rust]
+description = "Build the java binding rust code"
+script = '''
+#!/bin/bash
+set -ex
+cargo build -p risingwave_java_binding
+'''
+
+[tasks.build-java-binding-java]
+description = "Build the java binding java code"
+script = '''
+#!/bin/bash
+set -ex
+cd src/java_binding/java
+mvn clean package
+'''
+
+[tasks.run-java-binding-demo]
+description = "Run the java binding demo"
+dependencies = [
+    "build-java-binding-rust",
+    "build-java-binding-java"
+]
+script = '''
+#!/bin/bash
+set -ex
+RISINGWAVE_ROOT=.
+JAVA_BINDING_ROOT=${RISINGWAVE_ROOT}/src/java_binding
+java -cp ${JAVA_BINDING_ROOT}/java/java-binding/target/java-binding-1.0-SNAPSHOT.jar \
```

Comment:
Using `java` directly will cause the classpath to be set improperly, but it will be fixed in #7253.